### ISSUE TITLE
Zero probability not possible

### ISSUE THREAD

**Issue by arve0** [skratchdot/random-picker#3]
Hi!
I expected `picker.pick()` to never pick options with probability 0, it does:

```js
const Picker = require('random-picker').Picker
const picker = new Picker()
picker.option('null', 0)
picker.option('one', 1)
picker.pick()  // 50% chance of picking 'null'
assert(picker.totalScore() === 1)  // totalScore is 2
```

Demo: https://runkit.com/embed/xs8tjunjca7x

**Comment by arve0** [comment]
The error is here: https://github.com/skratchdot/random-picker/blob/master/index.js#L13

```js
0 || 1 // evaluates to 1
```

**Comment by skratchdot** [comment]
Fixed by PR #4
